helpers/freedom_helper.js

```diff
@@ -0,0 +1,66 @@
+'use strict';
+
+const Axios = require('axios');
+const https = require('https');
+const config = require('config/config');
+const accounts = require('freedom-accounts-util');
+
+const { rejectUnauthorized } = config.CUDL;
+const {
+    base_url,
+    client_id,
+    user_agent,
+    endpoints,
+    client_secret
+} = config.FREEDOM;
+
+const DASHBOARD_SCOPES = config.ACCOUNTS_API.scopes.DASHBOARD;
+
+
+module.exports = {
+    get_user_information,
+    get_oauth_access_token
+};
+
+function get_oauth_access_token(params) {
+    params.client_id = client_id;
+    params.client_secret = client_secret;
+
+    return accounts.generate_token(DASHBOARD_SCOPES.USER_READONLY)
+        .then(({access_token}) => {
+            const url = `${base_url}${endpoints.OAUTH_ACCESS_TOKEN}`;
+            const options = {
+                headers: {
+                    'Content-Type': 'application/json',
+                    'Authorization': `bearer ${access_token}`,
+                    'User-Agent': user_agent
+                },
+                httpsAgent: new https.Agent({
+                    rejectUnauthorized: rejectUnauthorized
+                })
+            };
+
+            return Axios.post(
+                url,
+                params,
+                options
+            );
+        });
+}
```

Comment:
Write this entirely as async/await. There's no point in writing in Promise syntax anymore since this would be the boilerplate and we expect to have our future code to have the latest syntax available.

Reply:
@wildtron I refactor the code and converted it to `async/await`.